src/sinopiaServer.js

```diff
@@ -154,3 +154,11 @@ export const publishRDFResource = async (currentUser, rdf, group) => {
   await authenticate(currentUser)
   return await instance.createResourceWithHttpInfo(group, rdf, { contentType: 'application/n-triples' })
 }
+
+export const updateRDFResource = async (currentUser, uri, rdf) => {
```

Comment:
would this new function warrant a test?

Reply:
This is only making two calls: authenticate and then the server.  I don't know that a test is going to be useful here other than to say: "Here are some mocks that show it's doing exactly what it's doing"

Reply:
Also, this is covered by the integration test.